README.md

```diff
@@ -4,3 +4,116 @@
 # MQTT bridge for Apache Kafka®
 
 This project provides a software component which acts as a bridge between [MQTT 3.1.1](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html) and an [Apache Kafka®](https://kafka.apache.org/) cluster.
+
+## Running the bridge
+
+### Locally
+
+Clone the repository and update your dependencies using maven.
```

Comment:
Sync the dependencies.

Reply:
I still don't get it, sorry. If you are referencing to download the dependency, it should be part of a section talking about "how to build" and running the corresponding maven command. It will be the one to download the dependency. It's not something the user has to do manually.

Reply:
this is out for now